src/bundle/Resources/views/themes/admin/ui/component/macros.html.twig

```diff
@@ -8,3 +8,18 @@
         {%- endif -%}
     {%- endfor -%}
 {%- endmacro -%}
+
+{% macro results_headline(count, has_filters = false, phrase = null) %}
+    {% trans_default_domain 'list' %}
```

Comment:
Do we want a new domain for these translations?

Reply:
yes please